quarkus-build-caching-extension/README.md

```diff
@@ -77,7 +75,13 @@ Add the `track-prod-config-changes` execution to the `quarkus-maven-plugin` conf
 </plugin>
 ```
 
-The generated `.quarkus/quarkus-prod-config-dump` file must be checked in the project repository to allow a cache hit on a first build after cloning.
+## Improving the cache hit rate
+The file `.quarkus/quarkus-prod-config-dump` generated by the Quarkus build `goal` must be present to allow a cache hit.
+
+In order to increase the cache hit rate, it is possible to check-in the file in the source code repository.
+in continuous integration environment, an alternative is to have a cache restoring the file.
```

Comment:
`an alternative is to have a cache restoring the file.`

The word cache here can be misleading. I assume you mean some sort of CI store/restore functionality? Not the build cache?

Reply:
Correct, I'll update with
> in continuous integration environment, an alternative is to have the file restored (see [this option](https://github.com/marketplace/actions/cache#restoring-and-saving-cache-using-a-single-action) for GitHub actions as an example).

Reply:
i like this